integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWDTModelNoServer.java

```diff
@@ -0,0 +1,356 @@
+      String managedServerPodName = managedServerPrefix + j;
+      String expectedStr = "Hello World, you have reached server " + MANAGED_SERVER_NAME_BASE + j;
+
+      logger.info("Checking that application is running on managed server pod {0}  in namespace {1} with "
+          + "expectedString {3}", managedServerPodName, domainNamespace, expectedStr);
+      checkAppIsRunning(
+          domainNamespace,
+          managedServerPodName,
+          expectedStr);
+    }
+
+    logger.info("Domain {0} is fully started in namespace {1} - servers are running and application is available",
+        domainUid, domainNamespace);
+
+    // delete domain and cluster
+    shutdownDomainAndVerify(domainNamespace, domainUid, replicaCount);
+    deleteDomainResource(domainNamespace, domainUid);
+    deleteClusterCustomResource(clusterName, domainNamespace);
+  }
+
+  /**
+   * Test in the WDT model file there is no AdminServerName and in Server section the server 'adminserver' defined.
+   * Note that the server name 'adminserver' is all lower case.
+   * Disabled due to bug.
+   */
+  @Disabled
+  @Test
+  @DisplayName("Test in the WDT model file there is no AdminServerName and in Server section 'adminserver' defined")
+  void testWdtModelNoAdminServerNameWithAdminServerLLowercaseInServer() {
+
+    String wdtModelFile = "model-noadminservername-hasadminserverlowercase.yaml";
+    imageName = createAndVerifyDomainImage(wdtModelFile);
+
+    createMiiDomainAndVerify(domainNamespace, domainUid,
+        imageName, adminServerPodName, managedServerPrefix, replicaCount);
+
+    //check and wait for the application to be accessible in all server pods
+    for (int j = 1; j <= replicaCount; j++) {
+      String managedServerPodName = managedServerPrefix + j;
+      String expectedStr = "Hello World, you have reached server " + MANAGED_SERVER_NAME_BASE + j;
+
+      logger.info("Checking that application is running on managed server pod {0}  in namespace {1} with "
+          + "expectedString {3}", managedServerPodName, domainNamespace, expectedStr);
+      checkAppIsRunning(
+          domainNamespace,
+          managedServerPodName,
+          expectedStr);
+    }
+
+    logger.info("Domain {0} is fully started in namespace {1} - servers are running and application is available",
+        domainUid, domainNamespace);
+
+    // delete domain and cluster
+    shutdownDomainAndVerify(domainNamespace, domainUid, replicaCount);
+    deleteDomainResource(domainNamespace, domainUid);
+    deleteClusterCustomResource(clusterName, domainNamespace);
+  }
+
+  /**
+   * Test in the WDT model file there is AdminServerName set and the named server is not in Server section.
+   * In the model file, the AdminServerName is set to 'new-admin-server'. In the Server section, there is no
+   * 'new-admin-server' set. There is a server 'admin-server' set instead.
+   * Verify WDT will create the named server 'new-admin-server' as in the AdminServerName.
```

Comment:
Here we have one admin pod new-admin-server and one managed pod  admin-server ?

Reply:
yes